content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -224,45 +256,32 @@ If there are errors that you don't understand, or if traces are reported as flus
 {{< /programming-lang >}}
 {{< programming-lang lang="python" >}}
 
-For more visibility, include `DD_TRACE_LOGGING_RATE=0`.
+Logs generated by the Python Tracer have the logging handler name [**ddtrace]**. 
```

Comment:
We already said this on line 33, should we delete it from there or here? Not sure, just pointing it out. Feel free to resolve if you think it's good to have in both places.

Reply:
I added it in two places in case someone jumped straight to the "Review" section. Which place would be better to explain the name? In the "how to enable" portion or the "review" portion?